internal/server/iigointernal/judiciary.go

```diff
@@ -81,11 +81,35 @@ func (j *judiciary) PayPresident() shared.Resources {
 	return hold
 }
 
-// InspectHistory checks all actions that happened in the last turn and audits them.
+// inspectHistory checks all actions that happened in the last turn and audits them.
 // This can be overridden by clients.
 func (j *judiciary) inspectHistory(iigoHistory []shared.Accountability) (map[shared.ClientID]roles.EvaluationReturn, bool) {
 	j.budget -= serviceCharge
-	return j.clientJudge.InspectHistory(iigoHistory)
+	success := false
+	tempResults := map[shared.ClientID]roles.EvaluationReturn{}
+	finalResults := getBaseEvalResults()
+	if j.clientJudge.HistoricalRetributionEnabled() {
+		for _, v := range j.localHistoryCache {
+			res, rsuccess := j.clientJudge.InspectHistory(v)
+			if !rsuccess {
+				success = false
```

Comment:
never used

Reply:
Oh crap yeah, not sure what I was thinking. thanks for the catch, patched in latest commit.